scripts/hail_batch/main.py

```diff
@@ -16,7 +16,7 @@
 
 batch = hb.Batch(name='generate PCA loadings', backend=service_backend)
 
-dataproc.hail_dataproc_job(
+dataproc.hail_dataproc_job(  # pylint: disable=unexpected-keyword-arg
```

Comment:
If you remove the quotes from `num_secondary_workers='100'`, you probably don't need this linter override?

```suggestion
dataproc.hail_dataproc_job(
```

Reply:
on line 25, there's an option for 'job_name', which is not in the latest analysis-runner release (but it is in the example I took it from).

Reply:
Thanks a lot for pointing that out. This should be fixed after updating to 1.4.0 of the analysis-runner (see https://github.com/populationgenomics/analysis-runner/pull/74).

You should still fix the type of the `num_secondary_workers` parameter to match the function declaration though.